cgmanifests/generated/cgmanifest.json

```diff
@@ -164,7 +164,7 @@
       "component": {
         "type": "git",
         "git": {
-          "commitHash": "3acac70a551c321574732e5bfd67930244bb7151",
+          "commitHash": "fc645b7626ebf86530dbd82fbece74d457e7ae07",
```

Comment:
for emsdk and onnx-tensorrt, the commits were updated to match the current submodule versions